docs/google.rst

```diff
@@ -136,17 +137,25 @@ google.batch.subnetwork                        Define the name of the subnetwork
 
 Process definition
 ------------------
-Processes can be defined as usual and by default the ``cpus`` and ``memory`` directives are used to find the cheapest machine
-type available at current location that fits the requested resources. If ``memory`` is not specified, 1GB of memory is allocated per cpu.
 
-The process ``machineType`` directive can be a list of patterns separated by comma. The pattern can contain a `*` to match
-any number of characters and `?` to match any single character. Examples of valid patterns: `c2-*`, `m?-standard*`, `n*`.
+Processes can be defined as usual. By default, the ``cpus`` and ``memory`` directives are used to find the cheapest machine
+type that is available at the current location and that fits the requested resources. If ``memory`` is not specified, 1GB of memory is allocated per CPU.
 
-Alternatively it can also be used to define a specific predefined Google Compute Platform `machine type <https://cloud.google.com/compute/docs/machine-types>`_
-or a custom machine type.
+The ``machineType`` process directive can be specified in any of the following ways:
 
-When Fusion is enabled, by default, only machine types that allow to attach local SSD disks will be used. If you specify your own
-machine type or machine series they should allow to attach local SSD disks, otherwise the job scheduling will fail.
+* A comma-separated list of patterns. The pattern can contain a ``*`` to match any number of characters
+  and ``?`` to match any single character. Examples of valid patterns: ``c2-*``, ``m?-standard*``, ``n*``.
+
+* A predefined Google Compute Platform `machine type <https://cloud.google.com/compute/docs/machine-types>`_ or `custom machine type <https://cloud.google.com/compute/docs/instances/creating-instance-with-custom-machine-type>`_.
+
+* An `instance template <https://cloud.google.com/compute/docs/instance-templates>`_, specified as ``template://<instance-template>``. Using an instance template
+  will overwrite the ``accelerator`` directive, as well as the following Google Batch options: ``cpuPlatform``, ``preemptible``, and ``spot``. To use an instance template
+  with GPUs, you must also set the ``google.batch.installGpuDrivers`` config option to ``true``. To use an instance template with Fusion, the instance template must include
+  a ``local-ssd`` disk with at least 375 GB. Requires version ``23.03.0.edge`` or later.
```

Comment:
@jordeu Do you think this will work? Fusion adds a local SSD through the instance policy, which is ignored if an instance template is specified. So I figured, if the instance template includes an SSD disk, then Fusion should still work.

Reply:
And the device name should be "fusion".

See here:
http://github.com/nextflow-io/nextflow/blob/a33a6f47e2160f476de9024b9415d63a57647479/plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchTaskHandler.groovy#L253-L253

And how then is used to make Google Batch automatically mount it inside the container:
http://github.com/nextflow-io/nextflow/blob/a33a6f47e2160f476de9024b9415d63a57647479/plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchFusionAdapter.groovy#L52-L52

Reply:
Okay, then it will work as long as the device name is `fusion`. The volume mounts are not in the instance policy so they will work the same.

Reply:
```suggestion
  with GPUs, you must also set the ``google.batch.installGpuDrivers`` config option to ``true``. To use an instance template with Fusion, the instance template must include
  a ``local-ssd`` disk named ``fusion`` with at least 375 GB. Requires version ``23.03.0.edge`` or later.
```